pkg/trtl/peers_test.go

```diff
@@ -122,7 +122,6 @@ func (s *trtlTestSuite) TestPeers() {
 	require.Equal(network[3].Addr, rep.Peers[0].Addr)
 	require.Equal(network[3].Name, rep.Peers[0].Name)
 	require.Equal(network[3].Region, rep.Peers[0].Region)
-	require.NotEqual(rep.Peers[0].Created, rep.Peers[0].Modified)
```

Comment:
The problem was that tests executed so fast that the created and modified timestamps were less than a second apart, which meant the test could fail occasionally since the timestamp is only to the second granularity.